CorsixTH/Lua/languages/english.lua

```diff
@@ -741,18 +776,18 @@ cheats_window = {
 tooltip.cheats_window = {
   close = "Close the cheats dialog",
   cheats = {
-    money = "Adds 10.000 to your bank balance.",
-    all_research = "Completes all research.",
-    emergency = "Creates an emergency.",
-    vip = "Creates a VIP.",
-    earthquake = "Creates an earthquake.",
-    epidemic = "Creates an contagious patient who may cause an epidemic to happen",
+    money = "Adds $10,000 to your bank balance",
```

Comment:
US vs UK I think?

Reply:
The default currency for the bank balance in the game shows a dollar ($) sign. Other texts in the game also refer to using dollars. As a result, I believe the English texts should folow that

Reply:
Failing that, it should at least be 10,000 rather than 10.000